src/components/dashboard/ProviderBlock.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import { Tag, Icon, Button, Alert } from 'rsuite';
-import firebase from 'firebase/app';
+import { FacebookAuthProvider, GoogleAuthProvider } from 'firebase/auth';
 import { auth } from '../../misc/firebase';
```

Comment:
See this for more ProviderBlock updates:

https://github.com/shelooks16/chat-app/commit/e066fb4a4162e71c0213dd4c158c9ef38e3e961e